components/brave_wallet/browser/brave_wallet_constants.h

```diff
@@ -36,6 +36,8 @@ constexpr char kRampID[] = "8yxja8782as5essk2myz3bmh4az6gpq4nte9n2gf";
 constexpr char kSardineStorefrontBaseURL[] = "https://crypto.sardine.ai";
 constexpr char kSardineClientTokensURL[] =
     "https://api.sardine.ai/v1/auth/client-tokens";
+constexpr char kTransakURL[] = "https://global.transak.com/";
+constexpr char kTransakApiKey[] = "985d14f0-4cf5-4a4c-8917-78107620d3b7";
```

Comment:
Hardcoding this API key is not a great idea as it allows anyone to impersonate requests as Brave. Is there any other options for authentication here such as a PKCE Auth flow?

Cc @fmarier who was recently looking at this class of issues who may have a better understanding off the top of his head.

Reply:
@kdenhartog The `apiKey` will still be shown to the user when the buy url is generated and opened in a new tab. I am not sure there is a way to avoid this from happening.